src/CopyrighFixer/CopyrighFixer/configparser.cpp

```diff
@@ -0,0 +1,50 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the lgplv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#include "configparser.h"
+#include <quasarapp.h>
+#include <iostream>
+
+
+namespace CopyrighFixer {
+ConfigParser::ConfigParser() {
+
+}
+
+const Config ConfigParser::parseOptions(const Config &conf) const {
+
+    Config configOption = conf;
```

Comment:
What you wan to do in this line ?
And for what you add the const Config &conf argument into the parseOptions method?